mobile/Screen/MapScreen.js

```diff
@@ -1,39 +1,265 @@
+    flex: 3,
+    borderRadius: 8,
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+  },
+  textContent: {
+    flex: 1,
+  },
+  cardTitle: {
+    fontSize: 12,
+    marginTop: 5,
+    fontWeight: 'bold',
+  },
+  cardDescription: {
+    fontSize: 12,
+    color: '#444',
+  },
+  markerWrap: {
     alignItems: 'center',
     justifyContent: 'center',
   },
+  marker: {
+    width: 20,
+    height: 20,
+    borderRadius: 5,
+    backgroundColor: 'rgba(130,4,150, 0.9)',
+  },
+  ring: {
+    width: 55,
+    height: 55,
+    borderRadius: 12,
+    backgroundColor: 'rgba(130,4,150, 0.3)',
+    position: 'absolute',
+    borderWidth: 1,
+    borderColor: 'rgba(130,4,150, 0.5)',
+  },
 });
 
-function MapScreen() {
-  const test = async () => {
-    console.log(URL);
-    try {
-      const result = await axios.post(`${URL}/test/post`, {
-        name: 'James', age: 20,
-      });
-      console.log(result.data);
-    } catch (err) {
-      console.error(err);
-    }
+const Images = [
+  { uri: 'http://overcomeandamplify.com/wp-content/uploads/2016/09/broadway-mall.jpg' },
+  { uri: 'https://img.ctykit.com/cdn/ca-dtla/images/tr:w-1800/king-eddy.jpg' },
+  { uri: 'https://images1.apartments.com/i2/KYlRC-2yNZXQSNh0JGMno_S-RIbHbS7bDROWLLtZ1uc/111/star-apartments-los-angeles-ca-primary-photo.jpg' },
+  { uri: 'https://www.lehrerarchitects.com/wp-content/uploads/2017/12/JAMES-WOOD_06-1920x1100.jpg' },
+];
+
+function GalleryScreen() {
+  const mapRef = useRef(null);
+  const state = {
+    markers: [
+      {
+        id: 1,
+        coordinate: {
+          latitude: 34.051060,
+          longitude: -118.247910,
+        },
+        title: 'Skid Row History Museum and LA Poverty Department',
+        description: '250 S Broadway | (213) 413-1077',
+        image: Images[0],
+      },
+      {
+        id: 2,
+        coordinate: {
+          latitude: 34.046070,
+          longitude: -118.247540,
+        },
+        title: 'Open Mic Night with Unkal Bean (King Eddy Saloon)',
+        description: '131 E 5th St.',
+        image: Images[1],
+      },
+      {
+        id: 3,
+        coordinate: {
+          latitude: 34.043580,
+          longitude: -118.247680,
+        },
+        title: 'Piece by Piece (Star Apartments)',
+        description: '240 E 6th St. | (323) 963-3372',
+        image: Images[2],
+      },
+      {
+        id: 4,
+        coordinate: {
+          latitude: 34.044536,
+          longitude: -118.244873,
+        },
+        title: 'Movies on the Nickel (James Wood Community Center',
+        description: '400 E 5th St. | (213) 229-9602',
+        image: Images[3],
+      },
+    ],
+    region: {
+      latitude: 34.0442,
+      longitude: -118.2439,
+      latitudeDelta: 0.005, // smaller value = more zoomed in
+      longitudeDelta: 0.005,
+    },
   };
 
+  const [mounted, setMounted] = useState(false);
+
+  if (!mounted) {
+    mapRef.index = 0;
+    mapRef.animation = new Animated.Value(0);
+  }
```

Comment:
Unsure what this does when `setMounted` is not used. I guess it doesn't matter too much but this causes an Eslint error